src/yunohost/domain.py

```diff
@@ -220,17 +220,20 @@ def domain_remove(auth, domain, force=False):
     msignals.display(m18n.n('domain_deleted'), 'success')
 
 
-def domain_dns_conf(domain):
+def domain_dns_conf(domain, ttl):
```

Comment:
You should set default value for *ttl* to `None`. As it's defined as an optional argument in the actions map, it allows to unify the action/method declaration in case of a Python call.